lms/templates/bookmarks/bookmarks_list.underscore

```diff
@@ -1,10 +1,12 @@
 <div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
```

Comment:
Nit: rename this file to `bookmarks-list.underscore` (`-` instead of `_`) to follow our underscore template naming conventions.